jaeger/injector/mutator/patch.go

```diff
@@ -1,6 +1,11 @@
 package mutator
 
 const tpl = `[
+  {
+    "op": "add",
+    "path": "/metadata/annotations/{{.Annotation}}",
```

Comment:
is it worth adding a template variable and dynamic json pointer escaping logic just to reuse the constant?  I'm all for reusing constants, but it might be simpler here to just hardcode it as:

```
"path": "/metadata/annotations/jaeger.linkerd.io~1tracing-enabled",
```

Reply:
Yea there are a few hoops to jump through here for something that is ultimately not going to change. I'll remove this from here as well as the tap-injector.